### PR TITLE
docs: add application.run to application lifecycle article

### PR DESCRIPTION
**Pull request by NickIliev**
Article and snippet revised with `application.run`

### CODE REVIEW

File: docs/core-concepts/application-lifecycle.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ import { Component } from '@angular/core';
 import { AppComponent } from './app.component';
 ```
 
-Some modules are libraries of other modules. Modules installed as npm packages (like `@angular/core` in the above example) should be referenced without a path prefix. When we import from one of our own files, we prefix the module name with the file path. In this example we specify a relative file path (./). That means the source module is in the same folder (./) as the module importing it. 
+Some modules are libraries of other modules. Modules installed as npm packages (like `@angular/core` in the above example) should be referenced without a path prefix. When we import from one of our own files, we prefix the module name with the file path. In this example, we specify a relative file path (./). That means the source module is in the same folder (./) as the module importing it.
```
Comment (ggarabedian) [inline docs/core-concepts/application-lifecycle.md]: "Some modules are libraries of other modules" sounds a bit strange. I know this is not written now, but you may want to consider changing it. For example: 
"Some (of the) modules are/can be dependencies of other modules."
"Some (of the) modules can depend on one or more separate modules."

File: docs/core-concepts/application-lifecycle.md [hunk 2]
```diff
@@ -139,17 +139,20 @@ This method is required only for iOS applications.
...
 iOS calls UIApplication and triggers the application main event loop.
 */
 
-import { start as applicationStart } from "application";
-applicationStart({ moduleName: "main-page" });
+import * as application from "tns-core-modules/application";
+application.run({ moduleName: "main-page" });
 ```
+
+> **Note:** Prior to version 4.0.0 all NativeScript application had single topmost `Frame` implicitly created by the `application.start()` method. With NativeScript 4.x.x and above, you can create multiple `Frame` instances using the `application.run()` method. More about the `Frame` API and navigation could be found in the [applicaiton lifecycle article](https://docs.nativescript.org/core-concepts/navigation)
```
Comment (ggarabedian) [inline docs/core-concepts/application-lifecycle.md]: application -> (...) could be found in the [**applicaiton** lifecycle article] (...)

**Review by ggarabedian — APPROVED** [review]
Only one small typo to fix. Looks good to me 👍

**Review by vakrilov — APPROVED** [review]

File: docs/core-concepts/application-lifecycle.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ import { Component } from '@angular/core';
 import { AppComponent } from './app.component';
 ```
 
-Some modules are libraries of other modules. Modules installed as npm packages (like `@angular/core` in the above example) should be referenced without a path prefix. When we import from one of our own files, we prefix the module name with the file path. In this example we specify a relative file path (./). That means the source module is in the same folder (./) as the module importing it. 
+Some of the modules can depend on one or more separate modules.. Modules installed as npm packages (like `@angular/core` in the above example) should be referenced without a path prefix. When we import from one of our own files, we prefix the module name with the file path. In this example, we specify a relative file path (./). That means the source module is in the same folder (./) as the module importing it.
```
Comment (MartoYankov) [inline docs/core-concepts/application-lifecycle.md]: Two dots after first sentence.

**Pull request merged**